### PR TITLE
[READY] Adds Departmental Emergency Response Teams

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

This Pull Request adds Departmental ERTs, who serve their own functions instead of being divided like ERTs usually are.
Due to their more focused nature, they get better tools than what a divided ERT member would get.
That is, things like:

- Engineering and Medical get Experimental Tools.
- Security gets a full webbing with tools. (Along with the Tactical Energy Gun which normal Sec ERT gets)
- Engineering and Medical Commanders get a special hardsuit with built-in SecHUDs.
- Engineering Commander gets a Combat RCD.
- Medical Commander gets a Combat Experimental Quantum Medicine Injector.
- Engineering ERT is more prepared (by wearing oxygen tank internals).
- Engineering ERT might have a 5% chance to have the urge to build a shed, provide advice to the crew about home construction along with the tools and materials to make a state of the art Home.
- Security Commander is robust (as expected of a Head of Security that arrived from CentCom), his armor blocks knockdowns from shoves, he's been outfitted with a thermal eyepatch, and his helmet also has the regular built-in SecHUDs display.
- Medical ERT (And its Commander) have been outfitted with special, insulated, Nitrile gloves for quick body carrying.
- Medical ERT Units have a 15% chance to spawn... differently. Either with ancient Rods with enchanted healing power or through being a Survivor of a Crashed Mothership.
- Mining ERT, with upgrades, advanced plasma cutters, special hardsuits, and Combat Jump Boots.
- Mining ERT Commander gets a Kinetic Crusher + trophies along with it. He has the HECK Suit palette.
- A Security ERT member has a small chance to be a synth. (And an even smaller chance to become a MILITARY one.)
- **HONK!** _They will arrive._

## Why It's Good For The Game

image
image
image
image
image

More ERT Variety is good. Ever wanted to call an ERT specifically to combat a virus/tons of dead crew/Mutinies without annihilating with a Deathsquad/Supermatter blowing up/All Miners are fucking dead/The Clown is fucking dead/Want to maxcap medbay and blame it on ERT? Well, now's the time, these guys are going to do it for you.

## Changelog
:cl:
add: Added Departmental ERTs for the Medical, Security, and Engineering departments.
/:cl:

### CODE REVIEW

**Comment by GuillaumePrata** [comment]
> NO TG Files edited baybee

My favorite words

**Comment by SgtHunk** [comment]
An accomplishment

**Comment by ErwinSnowden** [comment]
And we need a person who's effectively a validhunting role w/ an electrode taser and nostun? Especially with the ability for mods to authorize emergency response teams?

This doesn't even have a mining response team to deal with megafauna. Downvoted.

**Comment by SgtHunk** [comment]
Give me sprites and I add a Mining ERT

Also normal sec ERT already gets stun weapons lmao, this just makes it so whoever calls for ERT doesn't have to call a Deathsquad instead to solve security 'issues'

File: code/FulpstationCode/fulp_ert.dm [hunk 1]
```diff
@@ -0,0 +1,670 @@
...
+			return FALSE
+
+
+// A few other ERT-specific items
+
+// Medical
+
+/obj/item/clothing/gloves/color/latex/nitrile/infiltrator/medical
+	name = "insulated nitrile gloves"
+	desc = "Very pricy Specialized gloves, thicker than latex. Transfers intimate paramedic knowledge into the user via nanochips, along with providing insulation."
```
Comment (ATH1909) [inline code/FulpstationCode/fulp_ert.dm:618]:
```suggestion
	desc = "Very pricey specialized gloves, thicker than latex. Transfers intimate paramedic knowledge into the user via nanochips, along with providing insulation."
```

Comment (trerri) [reply code/FulpstationCode/fulp_ert.dm:618]: pricy is correct dumbass

**Pull request closed without merging**